code/modules/mob/hear_say.dm

```diff
@@ -9,6 +9,8 @@
 			//Or someone snoring.  So we make it where they won't hear it.
 		return
 
+	message = capitalize(message)
```

Comment:
Out of curiosity, how much, and in what manner, would this have reduced everything everyone knew and loved to a pile of scattered ashes?